### DISCUSSION TITLE
How to install pd-upscaler version?

### DISCUSSION THREAD

**Discussion by CynicalShrew** [praydog/REFramework#946] [General Questions]
Edit: Nevermind, managed to figure it out.

**Comment by sic6SaNdMaN** [comment]
And how did you do it?

**Comment by LSGeoff** [comment]
If you go to the actions link at the top

<img width="1327" height="138" alt="image" src="https://github.com/user-attachments/assets/3a812504-d1f6-44f3-bb7f-774916ebc7a9" />

Then click the branch text with the arrow

<img width="1694" height="498" alt="image" src="https://github.com/user-attachments/assets/ed4b3771-d945-444a-9a5c-bf05573e3d20" />

then click on the top link (in this case the CMakeLists text)

<img width="1692" height="393" alt="image" src="https://github.com/user-attachments/assets/22ea64ad-cb62-4851-a6db-ebbc18836ad3" />

You should see the files that match that branch

<img width="1870" height="1477" alt="image" src="https://github.com/user-attachments/assets/576eafef-f7c8-42d1-9d2a-3456f2032863" />

As long as you are logged in to your github account they should be downloadable (by clicking on the file text or using the download arrow/icon on the right side)

**Reply by Metatron-dev-stack** [reply]
it appears now that the files have expired, help please im trying to put dlss on resident evil 7

**Reply by AzDaSpaz** [reply]
yeah i keep getting expired aswell

**Reply by sic6SaNdMaN** [reply]
<img width="1056" height="597" alt="image" src="https://github.com/user-attachments/assets/b4972d1a-3b58-45f2-8329-0ca5f0c1547a" />

scroll down to "Artifacts":
<img width="1634" height="661" alt="image" src="https://github.com/user-attachments/assets/896ff11e-4b38-48eb-8b22-c24a2578da45" />

**Reply by AzDaSpaz** [reply]
Thank you very much for your help I appreciate it

**Reply by mattiasb1234** [reply]
Seems like that has expired too. Any other way to get an upscaler build?

**Reply by sic6SaNdMaN** [reply]
I uploaded the versions I have lying around.
https://drive.google.com/file/d/1AjnNMLfNjUxXaJgOlAFYyzSK3wqhNnk-/view?usp=sharing

**Reply by azooart** [reply]
goat

**Reply by mattiasb1234** [reply]
Thanks! :)

**Comment by mihkeh** [comment]
Finalmente o temporatlupscaler aparece mas diz que é incompativel! o que fiz de errado?

<img width="315" height="827" alt="Captura de tela 2025-12-17 154605" src="https://github.com/user-attachments/assets/34b83505-c321-45d6-921b-ffede7abb46e" />
